includes/definitions/teradici-pcoip.yaml

```diff
@@ -0,0 +1,11 @@
+os: teradici-pcoip
+text: 'Teradici PCoIP'
+type: workstation
```

Comment:
Sorry didn't spot this earlier. 

If you are adding a new type then this needs to go into includes/definitions.inc.php (see network, firewall, appliance as examples).